cmd/otel-allocator/target/target.go

```diff
@@ -27,23 +27,25 @@ type LinkJSON struct {
 }
 
 type Item struct {
-	JobName       string
-	Link          LinkJSON
-	TargetURL     string
-	Label         model.LabelSet
-	CollectorName string
+	JobName       string         `json:"-"`
+	Link          LinkJSON       `json:"-"`
+	TargetURL     []string       `json:"targets"`
+	Labels        model.LabelSet `json:"labels"`
```

Comment:
Is this a breaking change?

Reply:
This matches exactly what we were returning previously from the targetGroupJSON object, except we just return this rather than creating a new object.

Reply:
This would break someone who is importing this package, though. I will call that out in the PR description.

Reply:
Do you think I should keep the field name as `Label` instead or is it okay to make this change?

Reply:
Okay I see now that this is replacing `targetGroupJSON` (https://github.com/open-telemetry/opentelemetry-operator/pull/1293/files#diff-f7acbc836f7ddf2f8c66c36bbf55d831e7dbd492c606bde09215aa3979b000bdL31).  I think this is a good change 👍